### ISSUE TITLE
SQS 404 on posting to queue with trailing slash

### ISSUE THREAD

**Issue by yaitskov** [localstack/localstack#2874]
```
curl -X POST \
      -d 'Action=SendMessage&Version=2012-11-05&QueueUrl=http%3A%2F%2Flocalhost%3A4576%2F000000000000%2Fqueue_name%2F&MessageBody=m' \ 
      http://localhost:4576/
```

```
<ErrorResponse xmlns="http://queue.amazonaws.com/doc/2012-11-05/">
    <Error>
        <Type>Sender</Type>
        <Code>AWS.SimpleQueueService.NonExistentQueue</Code>
        <Message>The specified queue does not exist for this wsdl version.</Message>
        <Detail/>
    </Error>
    <RequestId>UBS0OKM9Y5ZYJCKBHHNW8U0SF52PU6NZ4BIZNDXCXT4I13E71M54</RequestId>
</ErrorResponse>
```

**Comment by j6k4m8** [comment]
I believe this is the same error as #2843. Not necessarily a dup, but seems to be the same underlying issue.

**Comment by yaitskov** [comment]
https://github.com/localstack/localstack/pull/2880

**Comment by whummer** [comment]
Thanks for reporting @yaitskov . This should be fixed in #2895 - can you please do a `docker pull localstack/localstack` and and give it another try with the latest image? Please report here if the problem persists. Thanks
